### DISCUSSION TITLE
lane2 edge image 검출 화면이 안나옵니다.

### DISCUSSION THREAD

**Discussion by jeongwon-Yang** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#182] [2024년도 교육 자료 관련 질문]
![image](https://github.com/user-attachments/assets/47e87244-2b14-4e69-9ca7-3725d263742f)

코드 실행 작동은 하나 아래 오류가 뜨면서 화면이 안나옵니다.

QObject::moveToThread: Current thread (0x5a1119305e90) is not the object's thread (0x5a1119451500).
Cannot move to target thread (0x5a1119305e90)

QObject::moveToThread: Current thread (0x5a1119305e90) is not the object's thread (0x5a1119451500).
Cannot move to target thread (0x5a1119305e90)

**Comment by HyeongKeunHong** [comment]
sudo pip uninstall opencv-python
sudo apt-get install libopencv-dev python-opencv

위 명령어 실행 후 노드 재실행 해보시기 바랍니다.
